config/twig_hooks/admin/invoice/show.yaml

```diff
@@ -16,9 +16,6 @@ sylius_twig_hooks:
         'sylius_admin.invoice.show.content.header.title_block.title':
             number:
                 template: '@SyliusInvoicingPlugin/admin/invoice/show/content/header/title_block/title/number.html.twig'
-                priority: 100
-            subtitle:
-                template: '@SyliusInvoicingPlugin/admin/invoice/show/content/header/title_block/title/subtitle.html.twig'
```

Comment:
Are we able to resolve this without removing and changing priorities of TH? Because this change will be a BC break 🤔